doofinder-for-woocommerce/includes/class-setup-wizard.php

```diff
@@ -312,6 +311,21 @@ private function register_ajax_action()
 			//$this->log->log('Doing ajax - doofinder_for_wc_check_data');
 			self::check_data();
 		});
+
+		add_action('wp_ajax_doofinder_set_connection_data', function () {
+			$this->log->log('Setup Wizard - Connect');
+			$this->process_step_2(true);
+			$resp = ["success" => Settings::is_api_configuration_complete()];
+
+			if (!empty($this->errors['wizard-step-2'])) {
+				$resp = [
+					"success" => false,
+					"errors" => $this->errors['wizard-step-2']
+				];
+			}
+
+			die(json_encode($resp));

```

Comment:
what is this die for?

Reply:
It is a way to ensure that the connection gets closed when doing an admin-ajax response